libraries/plugins/custom_operations/custom_operations_plugin.cpp

```diff
@@ -126,14 +128,25 @@ void custom_operations_plugin::plugin_set_program_options(
    boost::program_options::options_description& cfg
    )
 {
+   cli.add_options()
+         ("custom-operations-start-after-block", boost::program_options::value<uint32_t>(),
+          "Start processing custom operations transactions with the plugin only after this block(1)")
```

Comment:
`(1)` would be generated in the help/config if there is a default value assigned in the previous line.

Reply:
BTW I think `1` is not the best default value. For the mainnet, since the head block number is now `45446984`, I think `45000000` is a good default value. Usually node operators won't modify the default values when upgrading, so our decision is important.

Reply:
sounds good, changed at https://github.com/bitshares/bitshares-core/pull/2108/commits/4f219aa587a19e38445b532882c5a3d5159ab0a0